.github/workflows/build_test.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, linux]
```

Comment:
The build test seemed fine before with the Github runner. I checked between this pipeline run and the last one on the main branch, but the custom runner does not seem to speed up the build process. So I would propose to not adapt it for now.

Reply:
Sounds good for me :+1: